service-rpc/src/main/thrift/rpc.thrift

```diff
@@ -184,6 +184,7 @@ struct TSFetchMetadataResp{
 		8: optional set<string> devices
 		9: optional list<string> nodesList
 		10: optional map<string, string> nodeTimeseriesNum
+		11: optional set<string> childPaths
```

Comment:
It is not much related to this PR, but seeing this, I think maybe we should break down this Resp into several smaller ones so that it is more clear what it responds to.

Reply:
> It is not much related to this PR, but seeing this, I think maybe we should break down this Resp into several smaller ones so that it is more clear what it responds to.

Did you mean we can separate `TSFetchMetadataResp` into some ones like `TSFetchDevicesResp`, `TSFetchNodeListResp`, etc? 
That's a good point.

Reply:
Yes, besides you, other people are adding things into this structure, making it bigger and bigger. I don't think it is a good thing.